src/Cache/_ScsDataClient.php

```diff
@@ -38,7 +52,6 @@ class _ScsDataClient
     private static int $TIMEOUT_MULTIPLIER = 1000000;
```

Comment:
What does this do? :thinking:

Reply:
There's a comment right above this explaining it.